synbiohub_adapter/__init__.py

```diff
@@ -1,12 +1,2 @@
 # See https://www.python.org/dev/peps/pep-0440/ for info on version numbering
 __version__ = '1.1'
-
-from synbiohub_adapter.SynBioHubUtil import (
```

Comment:
This introduces a breaking change the the synbiohub_adapter public API. This will break scripts that use synbiohub_adapter. For example, this will break the sbh-metrics project that gathers metrics from the SD2 SynBioHub.

An example of code that will no longer work:

```
import synbiohub_adapter as sbha
sbha.SD2Constants.SD2_SERVER
```

That used to work, and now results in an exception:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'synbiohub_adapter' has no attribute 'SD2Constants'
```